d2l-sticky-element.html

```diff
@@ -0,0 +1,56 @@
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+    Copied with modifications and fixes from https://github.com/seaneking/sticky-element/blob/master/sticky-element.html
+-->
+
+<script src="../Stickyfill/dist/stickyfill.min.js"></script>
+<dom-module id="d2l-sticky-element">
+	<template>
+		<style>
+			:host {
+				position: -webkit-sticky;
+				position: sticky;
+				top: 0;
+			}
+		</style>
+		<content></content>
+	</template>
+	<script>
+		/* global Stickyfill */
+		document.addEventListener('DOMContentLoaded', Stickyfill.rebuild);
```

Comment:
It was in the original sticky-element. It probably isn't necessary because `_updateSticky` won't be called until the component is attached and painted.

`Stickyfill.rebuild` "Recalc all metrics and update stickies' positions."
https://github.com/wilddeer/stickyfill#stickyfillrebuild